web/src/components/pages/dashboard/dashboard.tsx

```diff
@@ -272,6 +274,14 @@ export default function Dashboard() {
           inviteId="#####"
           onRequestClose={() => {
```

Comment:
For this one, we want to check the achievement when "Copy link" is pressed.

See my long comment above for how I think this should integrate with the "sent an invite and contributed in the same session" achievement.

Reply:
Good point. Have done that now.

Reply:
Yep. Now we check the achievement when the user clicks the "Copy link" and shows the toast when the user closes the `invite modal`.